deploy/docker/fs/opt/newrelic/newrelic-template.yml

```
@@ -0,0 +1,400 @@
+# This file configures the New Relic agent.  New Relic monitors
+# Java applications with deep visibility and low overhead.  For more details and additional
+# configuration options visit https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file.
+#
+# <%= generated_for_user %>
+#
+# This section is for settings common to all environments.
+# Do not add anything above this next line.
+common: &default_settings
+
+  # ============================== LICENSE KEY ===============================
+  # You must specify the license key associated with your New Relic
+  # account. For example, if your license key is 12345 use this:
+  # license_key: '12345'
+  # The key binds your agent's data to your account in the New Relic service.
+  license_key: $APPSMITH_NEW_RELIC_APM_LICENSE_KEY
```

Change to literal block so we don't have to add `'` in the script when doing the replacement. Its simpler and safer in most cases.

```suggestion
  license_key: |
    $APPSMITH_NEW_RELIC_APM_LICENSE_KEY
```